jaeger/jaeger_install/rhbjaeger-deploying.adoc

```diff
@@ -1,42 +1,78 @@
 [id="deploying-jaeger"]
 = Deploying Jaeger
-include::modules/ossm-document-attributes.adoc[]
+include::modules/jaeger-document-attributes.adoc[]
 :context: jaeger-deploying
 
 toc::[]
 
-The Jaeger Operator includes a custom resource definition (CRD) file that determines the architecture and configuration settings to use when creating the Jaeger custom resources (CR). You can either install the default configuration or modify the custom resource file to better suit your business requirements.
+The Jaeger Operator includes a custom resource definition (CRD) file that defines the architecture and configuration settings for the Jaeger resources. You can either install the default configuration or modify the file to better suit your business requirements.
 
-Each Jaeger instance is associated with a deployment strategy. You define the strategy is defined in the custom resource file and when you create a Jaeger instance the Operator uses the deployment strategy and configuration options in the CR file to create the objects necessary for the deployment.
-The Jaeger Operator currently supports the following deployment strategies:
+You can deploy Jaeger following different deployment strategies. You define the strategy in the custom resource file. When you create a Jaeger instance the Operator uses this configuration file to create the objects necessary for the deployment.
```

Comment:
This seems out of order to me. 

I would delete "You can deploy Jaeger following different deployment strategies." And start with the following:

>When you create a Jaeger instance, the Operator uses a configuration file to create the objects necessary for the deployment. You can define the strategy in a custom resource file like the one in the following example.

Reply:
Changed to "Jaeger has pre-defined deployment strategies. You specify a deployment strategy in the custom resource file."